examples/opensora_hpcai/scripts/train.py

```diff
@@ -520,6 +525,44 @@ def main(args):
         ema=ema,
     )
 
+    if (args.mode == 0) and (args.bucket_config is not None):
+        video = ms.Tensor(shape=[None, None, 3, None, None], dtype=ms.float32)
+        caption = ms.Tensor(shape=[None, 200, 4096], dtype=ms.float32)
+        mask = ms.Tensor(shape=[None, 200], dtype=ms.uint8)
+        frames_mask = ms.Tensor(shape=[None, None], dtype=ms.bool_)
+        num_frames = ms.Tensor(
+            shape=[
+                None,
+            ],
+            dtype=ms.float32,
+        )
+        height = ms.Tensor(
+            shape=[
+                None,
+            ],
+            dtype=ms.float32,
+        )
+        width = ms.Tensor(
+            shape=[
+                None,
+            ],
+            dtype=ms.float32,
+        )
+        fps = ms.Tensor(
+            shape=[
+                None,
+            ],
+            dtype=ms.float32,
+        )
+        ar = ms.Tensor(
+            shape=[
+                None,
+            ],
+            dtype=ms.float32,
+        )
```

Comment:
To turn off ugly `black`'s formatting:
```suggestion
        # fmt: off
        num_frames = ms.Tensor(shape=[None, ], dtype=ms.float32)
        height = ms.Tensor(shape=[None, ], dtype=ms.float32)
        width = ms.Tensor(shape=[None, ], dtype=ms.float32)
        fps = ms.Tensor(shape=[None, ], dtype=ms.float32)
        ar = ms.Tensor(shape=[None, ], dtype=ms.float32)
        # fmt: on
```

Reply:
can just ust `ms.Tensor(shape=[None], dtype=ms.float32)` with add extra `# fmt: on`

Reply:
True. I think `[None, ]` is just used to indicate that the array is 2D and the batch dimension is variable.

Reply:
no it is 1d